src/tk/widget/input.h

```diff
@@ -42,7 +42,7 @@ void window_setInputCallback( const unsigned int wid, char* name, void (*fptr)(u
 
 /* Filter constants. */
 #define INPUT_FILTER_NUMBER     "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ[]{}()-=*/\\'\"~<>!@#$%^&|_`"
-#define INPUT_FILTER_RESOLUTION "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ[]{}()-=*/\\'\"~<>!@#$%^&|_`"
+#define INPUT_FILTER_RESOLUTION "abcdefghijklmnopqrstuvwyzABCDEFGHIJKLMNOPQRSTUVWXYZ[]{}()-=*/\\'\"~<>!@#$%^&|_`"
```

Comment:
I don't understand this, why remove `x`?

Reply:
This is for screen resolutions like "1280x720" -- also, it's what options.c had before I moved *and* broke that filter.

Reply:
Aha! Thanks